src/main.ts

```diff
@@ -0,0 +1,8 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+  const app = await NestFactory.create(AppModule);
```

Comment:
Also we are planning to use single domain for both web and API.. so lets do like this:

app.setGlobalPrefix('api');

we can handle request forwarding with reverse proxy